cmd/podman/create.go

```diff
@@ -244,18 +244,15 @@ func loadAppArmor(config *cc.CreateConfig) error {
 	return nil
 }
 
-func parseSecurityOpt(config *cc.CreateConfig, securityOpts []string) error {
-	var (
-		labelOpts []string
-		err       error
-	)
+func parseSecurityOpt(config *cc.CreateConfig, securityOpts []string) ([]string, error) {
```

Comment:
I still don't think this is right - the AppArmor and seccomp portions of the function should never happen if we are privileged

Reply:
Woops, sorry forgot to add the privileged check back. Should be fixed now.

Reply:
Why aren't you just doing this inside of parseSecurityOpt.  Just do the config.Privileged check their and set the process label to ""?

Reply:
But we don't want to call parseSecurityOpt if privileged is set right as we might turn on seccomp and apparmor. Hence the check to ensure we are not privileged, call parseSecurityOpts. And if we are privileged initialize the labels anyways and if privileged set process label to "".